src/algorithms/calorimetry/TrackClusterMergeSplitter.cc

```diff
@@ -0,0 +1,460 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Derek Anderson
+
+#define EICRECON_TRACKCLUSTERMERGESPLITTER_CC
+
+#include <cmath>
+#include <regex>
+#include <utility>
+#include <algorithm>
+// dd4hep utilities
+#include <DD4hep/Readout.h>
+// jana utilities
+#include <JANA/JException.h>
```

Comment:
```suggestion
```
Not supposed to use JANA things in algorithms.

Reply:
Whoops! Good catch!